

class CustomErrors(Exception):

    def __init__(self, message: str):
        self.message = message

    def get_message(self):
        return self.message